README.md

```diff
@@ -53,6 +53,9 @@ OPTIONS:
     -t, --threshold <THRESHOLD>
             Sets the maximum amount of space to be used for Docker images (default: 10 GB)
 
+    -m, --min-age <MIN AGE>
+            Sets the minimum age of images to be considered for deletion
+
```

Comment:
I just checked out your branch and ran `cargo -- --help` and it puts this new option right after the `--keep` option. Please make this _exactly_ match the help text output.

Reply:
Done